tensornetwork/quantum/quantum.py

```diff
@@ -264,6 +261,12 @@ def tensor_prod(self, other: "QuOperator"):
 
     return quantum_constructor(out_edges, in_edges, ref_nodes, ignore_edges)
 
+  def outer_product(self, other: "QuOperator"):
+    """Outer product with another operator.
+    Synonym for `tensor_product()`.
+    """
+    return self.tensor_product(other)
+
```

Comment:
I don't think unneeded aliases are good. Just pick one, I have no preference at this point.

Reply:
Ok.